### ISSUE TITLE
Upgrade @types/zen-observable to 0.5.4

### ISSUE THREAD

**Issue by ChenRoth** [apollographql/apollo-link#650]
current version conflicts `@types/node`'s newly added `readonly` modifier for the `observable` type
Referencing this issue: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/26239#issuecomment-395925181

**Comment by lukeify** [comment]
In the meantime, if you're using yarn, use the `resolutions` block in package.json to force 0.5.4 for zen-observable.

```
"resolutions": {
    "@types/zen-observable": "0.5.4"
}
```

**Comment by JohannesHoppe** [comment]
This should be merged with highest priority!

Meanwhile:
```
npm i @types/zen-observable@0.5.4
```
did the same trick for me (NPM user).

**Comment by BPScott** [comment]
Fixed in #681.

Just waiting on a release of apollo-link-polling and apollo-link-retry

**Comment by JoviDeCroock** [comment]
This is fixed so closing this
